src/Persistence/abstractconfigstore.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+
+class ConfigStore(ABC):
```

Comment:
is there a way to define this class as abstract?
Or is there no such thing as an 'interface' in python?

Reply:
Python doesn't have interfaces, but can approximate some of the behaviour through Abstract Base Class ("ABC"): https://docs.python.org/3/library/abc.html